.github/CONTRIBUTING.md

```diff
@@ -258,9 +251,9 @@ This prevents nesting levels from getting deeper then they need to be.
 	* Areas should not be var-edited on a map to change it's name or attributes. All areas of a single type and it's altered instances are considered the same area within the code, and editing their variables on a map can lead to issues with powernets and event subsystems which are difficult to debug.
 
 ### User Interfaces
-* All new player-facing user interfaces must use TGUI-next; TGUI is deprecated. 
+* All new player-facing user interfaces must use TGUI-next; TGUI is deprecated.
```

Comment:
TGUI is ded pls restart

Reply:
You can remove the TGUI is deprecated part completely as it would be very difficult for someone to re-add tgui and PR an interface using it

Reply:
this can be done in their PR

Reply:
its already been merged @optimumtact